.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
   - conda create --yes -n env_name python=$PYTHON_VERSION pip numpy scipy nose flake8
   - source activate env_name
   - pip install coveralls
+  - pip install https://github.com/biocore/burrito-fillings/archive/master.zip
```

Comment:
I think the *real* reason is because although MOI is a required
dependency i.e. Qitta cannot be started without it, setuptools (the
underlying framework powering pip) is about to deprecate using arbitrary
download links in setup.py files.  Otherwise you would have to add
--allow-unverified --allow-something-else (flags that will soon be
removed) to the setup.py execution.

On (Mar-30-15|12:50), Antonio Gonzalez wrote:
> > @@ -12,6 +12,7 @@ install:
> >    - conda create --yes -n env_name python=$PYTHON_VERSION pip numpy scipy nose flake8
> >    - source activate env_name
> >    - pip install coveralls
> > +  - pip install https://github.com/biocore/burrito-fillings/archive/master.zip
> 
> travis: for testing.
> setup.py: for installing.
> 
> qiita: We have it in the instructios:
> https://github.com/biocore/qiita/blob/master/INSTALL.md
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/deblur/pull/19/files#r27425114